data/keys

```diff
@@ -39,20 +39,27 @@ Global {
 	KeyPress = "Ctrl Mod1 Shift Right" { Actions = "SendToWorkspace Prev; GoToWorkspace Prev" }
 	KeyPress = "Ctrl Mod1 Shift Up" { Actions = "SendToWorkspace NextV; GoToWorkspace NextV" }
 	KeyPress = "Ctrl Mod1 Shift Down" { Actions = "SendToWorkspace PrevV; GoToWorkspace PrevV" }
-	KeyPress = "Mod4 F1" { Actions = "SendToWorkspace 1" }
-	KeyPress = "Mod4 F2" { Actions = "SendToWorkspace 2" }
-	KeyPress = "Mod4 F3" { Actions = "SendToWorkspace 3" }
-	KeyPress = "Mod4 F4" { Actions = "SendToWorkspace 4" }
-	KeyPress = "Mod4 F5" { Actions = "SendToWorkspace 5" }
-	KeyPress = "Mod4 F6" { Actions = "SendToWorkspace 6" }
-	KeyPress = "Mod4 F7" { Actions = "SendToWorkspace 7" }
-	KeyPress = "Mod4 F8" { Actions = "SendToWorkspace 8" }
-	KeyPress = "Mod4 F9" { Actions = "SendToWorkspace 9" }
+	KeyPress = "Mod4 Shift 1" { Actions = "SendToWorkspace 1" }
+	KeyPress = "Mod4 Shift 2" { Actions = "SendToWorkspace 2" }
+	KeyPress = "Mod4 Shift 3" { Actions = "SendToWorkspace 3" }
+	KeyPress = "Mod4 Shift 4" { Actions = "SendToWorkspace 4" }
+	KeyPress = "Mod4 Shift 5" { Actions = "SendToWorkspace 5" }
+	KeyPress = "Mod4 Shift 6" { Actions = "SendToWorkspace 6" }
+	KeyPress = "Mod4 Shift 7" { Actions = "SendToWorkspace 7" }
+	KeyPress = "Mod4 Shift 8" { Actions = "SendToWorkspace 8" }
+	KeyPress = "Mod4 Shift 9" { Actions = "SendToWorkspace 9" }
 	# Simple window management
+	KeyPress = "Mod4 Shift V" { Actions = "Toggle Maximized False True" }
 	KeyPress = "Mod4 M" { Actions = "Toggle Maximized True True" }
+	KeyPress = "Mod4 Shift U" { Actions = "SetGeometry 50%x50% current HonourStrut" }
+	KeyPress = "Mod4 Shift Z" { Actions = "SetGeometry 50%x100% current HonourStrut" }
```

Comment:
hmm.. this works well with Windows-Shift-Left/Right, I guess. But whatabout 100%x50% to work with up/down?

Reply:
Yes, that would also be a possibility to have two wide windows above each other. I simply didn't add that because I don't use it. But I can include it if you want.